settings.gradle.kts

```diff
@@ -8,3 +10,22 @@ include(
     "samples:gradle-export-api",
     "test_projects:android"
 )
+
+plugins {
+    id("com.gradle.enterprise") version "3.4.1"
```

Comment:
could you move 
"com.gradle.enterprise" to `Plugins`
and 
"3.4.1" to `Versions`

Reply:
buildSrc is not available in the `settings.gradle` file.